lighthouse-core/scripts/roll-to-devtools.sh

```diff
@@ -53,6 +50,6 @@ rsync -avh "$lh_locales_dir" "$fe_locales_dir" --exclude="*.ctc.json" --delete
 echo -e "$check Locale JSON files copied."
 
 echo ""
-echo "Done. To rebase the test expectations, run: "
+echo "Done. To rebase the test expectations, run (note: you must use the integrated checkout): "
 echo "    yarn --cwd ~/chromium/src/third_party/devtools-frontend/src test 'http/tests/devtools/lighthouse/*.js' --layout-tests-dir test/webtests --reset-results"
```

Comment:
this doesn't seem to match the new hardcoded one above?

Reply:
The one above is the standalone checkout. This one needs to be buried within a chromium checkout (layout tests require chromium infra)